### DISCUSSION TITLE
How can I find SeisSol/Examples/tpv105/mirrorMesh.py

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#563] [Q&A] [answered]
!!! I am sorry, I have solved this problem, but I do not kown how to delete this discussion.
when I run ./generating_the_mesh.sh, I get this problem.
![image](https://user-images.githubusercontent.com/104721358/171554313-96c24cee-f383-4b2e-a54e-f8f738ac0ca5.png)

**Comment by daisy20170101** [comment, marked as answer]
https://github.com/SeisSol/Meshing/blob/master/mirrorMesh/mirrorMesh.py

**Reply by smallriver666** [reply]
> https://github.com/SeisSol/Meshing/blob/master/mirrorMesh/mirrorMesh.py

thank you very much, I have found it,  I am embrassed that I do not kown how to delete this discussion.

**Reply by smallriver666** [reply]
> https://github.com/SeisSol/Meshing/blob/master/mirrorMesh/mirrorMesh.py

and do you kown how can I find this module
![image](https://user-images.githubusercontent.com/104721358/171559610-8b179e4f-12fe-4f93-86e9-6e5e84c9b8f2.png)
I can see in this file import this module, but that can not find. Thank you.
![image](https://user-images.githubusercontent.com/104721358/171560143-834b1523-aba8-476e-9253-880936daf8a8.png)

**Reply by daisy20170101** [reply]
You can install it in python environment 
pip install seissolxdmf
